### ISSUE TITLE
[macOS] - Quit app with new session option

### ISSUE THREAD

**Issue by ghost** [aircrack-ng/aircrack-ng#1983]
# Issue type

- Defect - New Session

# System information

- OS: macOS 10.14
- CPU: Intel

# Aircrack-ng version

- Version: 1.4

# Defect
## How to reproduce the issue

When exec the following command.
```shell
aircrack-ng -a 2 --new-session mysession.session -w pass.dict -e my-essid -b 00:11:22:33:AA:BB:CC psk*.cap
```
I can't use new  session flag, program not run and quit.

**Comment by aircrack-ng** [comment]
1. Anything displayed by that command?
2. Is the network selected WPA and has handshakes? It may be related to https://github.com/aircrack-ng/aircrack-ng/issues/1978#issuecomment-437453244

**Comment by soareswallace** [comment]
In my case

1. No,  it was showing only this message: `Assertion failed: (ap_cur->wpa.state > 0), function main, file aircrack-ng.c, line 6738`.
 - Now I'm receiving `Packets contained no EAPOL data; unable to process this AP.`
2. Happen with WPA(0 handshakes) or No Data - WEP or WPA

**Comment by aircrack-ng** [comment]
6dea311 fixed the issue you mentioned in 1 (#1978).

Is what you're getting now incorrect? Could you zip and upload the PCAP?

**Comment by aircrack-ng** [comment]
2. Might be related to #1997 

Closing this ticket, assuming it's solved. If not, reopen with answers to the questions above
